bundles/org.openhab.binding.bticinosmarther/README.md

```diff
@@ -0,0 +1,328 @@
+# BTicino Smarther Binding
+
+The Smarther binding implements a bridge to the Legrand/BTicino Smarther v2.0 API and allows you to control your BTicino Smarther chronothermostat units with openHAB, making it possible to discover Smarther units connected to your Legrand/BTicino Smarther account.
+
+Smarther chronothermostat is produced by BTicino (https://www.bticino.com/products-catalogue/smarther-the-connected-thermostat/), has its own API set and doesn't support the OpenWebNet protocol.
+
+Companion HABPanel widget can be found here: [BTicino Smarther HABPanel widget](https://github.com/MrRonfo/openhab-smarther-widget)
+
+# Configuring the binding
```

Comment:
Please structure the README along our [template](https://github.com/openhab/openhab-core/blob/master/tools/archetype/binding/src/main/resources/archetype-resources/README.md) and use the same headers, if possible - also make sure to capitalize all headers.

Reply:
Sure. In this case, what paragraph is best suitable to host the Bridge configuration process (for me to address the next suggestion too) ?